### PR TITLE
Textarea

### PR DESCRIPTION
**Pull request by evgenoid**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the new behavior?
Added textarea component

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information

screencapture-localhost-9009-2023-08-22-15_41_42

### CODE REVIEW

File: src/form/Textarea/Textarea.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+  return (
+    <div
+      className={classNames(css.root, {
+        [css.fullWidth]: fullWidth,
+        [css.error]: error,
+        [css[size]]: size
+      })}
+    >
+      <textarea
```
Comment (amcdnl) [inline src/form/Textarea/Textarea.tsx:54]: Let's use the auto size text area.

File: src/utils/Theme/themes/dark.ts [hunk 2]
```diff
@@ -273,6 +273,18 @@ export const darkTheme: Theme = {
       'input-adornment-size': fontSizes.md,
       'input-adornment-fill': 'var(--input-color)'
     },
+    textarea: {
```
Comment (amcdnl) [inline src/utils/Theme/themes/dark.ts:276]: Need to add the tokens in the Theme example docs.

File: src/form/Textarea/Textarea.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+  /**
+   * If true, the field will take up the full width of its container.
+   */
+  fullWidth?: boolean;
+
+  /**
+   * Size of the field.
+   */
+  size?: 'small' | 'medium' | 'large';
```
Comment (amcdnl) [inline src/form/Textarea/Textarea.tsx:26]: @steppy452 - Do u think we should also adjust fonts w/ this?

**Review by amcdnl — CHANGES_REQUESTED** [review]

File: src/form/Textarea/Textarea.tsx (hunk 3 above)
Comment (evgenoid) [reply src/form/Textarea/Textarea.tsx:26]: I think no. At least in other UI libraries (MUI) font is still the same

File: src/form/Textarea/Textarea.tsx (hunk 1 above)
Comment (evgenoid) [reply src/form/Textarea/Textarea.tsx:54]: Optional?

Comment (amcdnl) [reply src/form/Textarea/Textarea.tsx:54]:
```
import ReactTextarea, { TextareaAutosizeProps } from 'react-textarea-autosize';
```

File: src/form/Textarea/Textarea.tsx (hunk 3 above)
Comment (steppy452) [reply src/form/Textarea/Textarea.tsx:26]: no, i don't think i've ever seen the font sizes change with the field size

**Review by amcdnl — APPROVED** [review]

**Comment by steppy452** [comment]
#16

**Pull request merged**